README.md

```diff
@@ -96,7 +129,7 @@ result = init.compute(X, daal4py.my_procid())
 d4p.daalfini()
```

Comment:
I think we can remove this portion about distributed algos.
Because it's not Scikit-learn patching.

Reply:
Done